### PR TITLE
Moves unshipped NuGet 6.2 APIs to PublicAPI.Shipped.txt files

### PR DESCRIPTION
**Pull request by dominoFire**
## Bug

Fixes: https://github.com/NuGet/Client.Engineering/issues/1617

Regression? Last working version: N/A

## Description

Ran `tools-local\ship-public-apis` tool to mark unshipped APIs as shipped in NuGet 6.2 release.

## PR Checklist

- [x] PR has a meaningful title
- [x] PR has a linked issue.
- [x] Described changes

- **Tests**
  - [ ] Automated tests added
  - **OR**
  
  - [ ] Test exception
  - **OR**
  - [x] N/A: Engineering procedure

- **Documentation**
  
  - [x] Documentation PR or issue filled: https://github.com/NuGet/docs.microsoft.com-nuget/pull/2744
  - **OR**
  - [ ] N/A

### CODE REVIEW

File: src/NuGet.Core/NuGet.Build.Tasks.Pack/PublicAPI.Shipped.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+NuGet.Build.Tasks.Pack.IPackTaskLogic.GetPackArgs(NuGet.Build.Tasks.Pack.IPackTaskRequest<NuGet.Commands.IMSBuildItem> request, NuGet.Commands.PackCommand.WarningPropertiesCollection warningPropertiesCollection) -> NuGet.Commands.PackArgs
```
Comment (nkolev92) [inline src/NuGet.Core/NuGet.Build.Tasks.Pack/PublicAPI.Shipped.txt]: I think this is something that should have probably been removed.

UsePublicApiAnalyzer is false for this project. 

Probably got commited mistakenly by someone.

**Review by nkolev92 — COMMENTED** [review]
One small change.

File: src/NuGet.Core/NuGet.Build.Tasks.Pack/PublicAPI.Shipped.txt (hunk 1 above)
Comment (dominoFire) [reply src/NuGet.Core/NuGet.Build.Tasks.Pack/PublicAPI.Shipped.txt]: Should I update `tools-local\ship-public-apis` tool to prevent this in the future?

Comment (dominoFire) [reply src/NuGet.Core/NuGet.Build.Tasks.Pack/PublicAPI.Shipped.txt]: Removed for now.

Comment (nkolev92) [reply src/NuGet.Core/NuGet.Build.Tasks.Pack/PublicAPI.Shipped.txt]: I don't think we ned to update the tooling. 
This file was committed by mistake, and the tooling works well enough.

**Review by nkolev92 — APPROVED** [review]

**Pull request merged**